sdk/core/System.ClientModel/api/System.ClientModel.netstandard2.0.cs

```diff
@@ -70,6 +70,20 @@ protected ContinuationToken(System.BinaryData bytes) { }
         public static System.ClientModel.ContinuationToken FromBytes(System.BinaryData bytes) { throw null; }
         public virtual System.BinaryData ToBytes() { throw null; }
     }
+    public partial class Credential
+    {
+        public Credential(string token, string tokenType, System.DateTimeOffset expiresOn) { }
+        public System.DateTimeOffset ExpiresOn { get { throw null; } protected set { } }
+        public System.DateTimeOffset? RefreshOn { get { throw null; } protected set { } }
+        public string Token { get { throw null; } protected set { } }
```

Comment:
should token be BinaryData? Isn't that what the code needs in majority of scenarios?

Reply:
It's almost always just a base64 encoded string - is there a type of data you had in mind that would benefit from being BinaryData?

Reply:
what do users do with the base64 string?

Reply:
Tokens are typically opaque to users - they are usually only interpreted by the service. In some cases they can even be encrypted.

Reply:
Yes, but still what would the caller of this getter do with the string? e.g. concatenate it with "Bearer : ", or do some more elaborate transformation,

Reply:
For code that directly interacts with authorization headers etc., the most elaborate transformation would be to concatenate it with the `TokenType` property's value, which is typically "Bearer".  Do you have some ideas about how `BinaryData` helps here?

Reply:
if it's just concatenation, then string is fine.